public/app/features/plugins/admin/api.ts

```diff
@@ -19,19 +19,27 @@ export async function getPluginDetails(id: string): Promise<CatalogPluginDetails
   const local = localPlugins.find((p) => p.id === id);
   const dependencies = local?.dependencies || remote?.json?.dependencies;
 
-  return {
+  const a = {
+    isDeprecated: remote?.status === 'deprecated',
     grafanaDependency: dependencies?.grafanaDependency ?? dependencies?.grafanaVersion ?? '',
     pluginDependencies: dependencies?.plugins || [],
     links: local?.info.links || remote?.json?.info.links || [],
     readme: localReadme || remote?.readme,
     versions,
   };
+
+  console.log('***', a);
```

Comment:
remove Console.log

Reply:
Thanks, good catch - will resolve this, as it has been removed by Marcus's suggestion.